### PR TITLE
Update changelog for next release

### PR DESCRIPTION
**Pull request by rnystrom**
Resetting the changelog for next hotfix release.

### CODE REVIEW

**Comment by jessesquires** [comment]
For some clarity: [JSQMessages](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/CHANGELOG.md) is not really a great example here. It's like this because I simply didn't have a CHANGELOG for a long time. [JSQCoreDataKit](https://github.com/jessesquires/JSQCoreDataKit/blob/develop/CHANGELOG.md) is a better example. 😄

**Comment by jessesquires** [comment]
Oh, also, want to make this `2.1.0`? 

I want to get this macOS support merged and fixed up, then released. We can tack-on any bug fixes, too.

https://github.com/Instagram/IGListKit/milestone/2

**Comment by rnystrom** [comment]
Oh, ok! Sgtm.

File: CHANGELOG.md [hunk 1]
```diff
@@ -51,7 +56,7 @@ You can find a [migration guide here](https://instagram.github.io/IGListKit/migr
 
 - Added support for cells created from nibs. [Sven Bacia](https://github.com/svenbacia) [(#56)](https://github.com/Instagram/IGListKit/pull/56)
 
-- Added an additional initializer for `IGListSingleSectionController` to be able to support single sections created from nibs. An example can be found [here](https://github.com/Instagram/IGListKit/tree/master/Examples/Examples-iOS/IGListKitExamples/ViewControllers/SingleSectionViewController.swift). [(#56)](https://github.com/Instagram/IGListKit/pull/56)
```
Comment (jessesquires) [inline CHANGELOG.md]: need to revert this change. (should use full URL)
